src/root/body/index.tsx

```diff
@@ -117,6 +117,14 @@ const Body: React.FC<params> = (props) => {
     }
   };
 
+  const getBackendUrl = (): string => {
```

Comment:
not sure we want to wait upon a call to the backend to fail; we would want to detect if the backend is configured and working (with a new /ping endpoint) as soon as the frontend comes up. We can postpone that in a future issue and have @CFIALeronB  work on it.

Reply:
https://github.com/ai-cfia/nachet-frontend/issues/55